diffs/preTestArbitrumUpdate20230503_postTestArbitrumUpdate20230503.md

```diff
@@ -0,0 +1,30 @@
+## Reserve changes
+
+### Reserves altered
+
+#### EURS ([0xD22a58f79e9481D1a88e00c343885A588b34b68B](https://https://arbiscan.io/address/0xD22a58f79e9481D1a88e00c343885A588b34b68B))
+
+| description | value before | value after |
+| --- | --- | --- |
+| supplyCap | 60,000 EURS | 65,000 EURS |
+| borrowCap | 45,000 EURS | 65,000 EURS |
```

Comment:
isn't current borrow cap 40,000?

Reply:
nvm, looks like 45,000 on-chain - do we update the text in forum post in this case?